extension/content/tests/components/recipes/RecipeListing.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { cleanup } from "@testing-library/react";
+import React from "react";
+
+import RecipeListing from "devtools/components/recipes/RecipeListing";
+
+import {
+  approvalRequestFactory,
+  revisionFactory,
+  recipeFactory,
+} from "devtools/tests/factories/recipes";
+
+afterEach(async () => {
+  jest.clearAllMocks();
+  await cleanup();
+});
+
+describe("RecipeListing", () => {
+  it("should have pending review tag when no approval", () => {
+    const approvalRequest = approvalRequestFactory.build({ approved: null });
+    const revision = revisionFactory.build(
+      { approval_request: approvalRequest },
+      { actionName: "branched-addon-study" },
+    );
+    const recipe = recipeFactory.build();
+    recipe.latest_revision = revision;
+
+    /* global renderWithContext */
```

Comment:
`renderWithContext` is something we injected globally in Jest tests, right? Can you add this to `extension/content/tests/.eslintrc.js` instead of putting it one-off here?